specification/cognitiveservices/OpenAI.Inference/models/transcription.create.tsp

```diff
@@ -0,0 +1,53 @@
+import "@typespec/rest";
+import "@typespec/http";
+
+using TypeSpec.Rest;
+using TypeSpec.Http;
+
+namespace Azure.OpenAI;
+
+@doc("""
+Transcription request.
+Requesting format 'json' will result on only the 'text' field being set.
+For more output data use 'verbose_json.
+""")
+model AudioTranscriptionsOptions {
```

Comment:
This is almost laughably minor, but it did come up as a "hmm, that's weird" for me when working with generated code, so it's worth bringing up even if it *is* laughable: I think we can use `AudioTranscription*` (singular, no s) rather than `AudioTranscriptions*` (with the s) as the base identifier. The main difference between this API and completions/chat/embeddings is the refreshing lack of an `N` equivalent; given there's no conceptual container like "choices" to account for, I think it'll fit nicely to provide `AudioTranscriptionOptions` to `getAudioTranscription` in the process of geting an `AudioTranscription` response result value.